src/MongrationDotNet/ClientSideDocumentMigration.cs

```diff
@@ -29,7 +28,10 @@ public override async Task ExecuteAsync(IMongoDatabase database, ILogger logger)
             IEnumerable<BsonDocument> documents;
             do
             {
-                documents = (await collection.Find(SearchFilters).ToListAsync()).Skip(updated).Take(BatchSize);
+                documents = PageThroughAllFilteredDocuments 
```

Comment:
Suggest we log the number of updated documents at the end as well.

Reply:
Done